Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -135,16 +135,15 @@ public override void ViewDidLayoutSubviews()
 			UpdateToolBarVisible();
 
 			var navBarFrame = NavigationBar.Frame;
-
 			var toolbar = _secondaryToolbar;
 			// Use 0 if the NavBar is hidden or will be hidden
 			var toolbarY = NavigationBarHidden || !NavigationPage.GetHasNavigationBar(Current) ? 0 : navBarFrame.Bottom;
 			toolbar.Frame = new RectangleF(0, toolbarY, View.Frame.Width, toolbar.Frame.Height);
 
 			double trueBottom = toolbar.Hidden ? toolbarY : toolbar.Frame.Bottom;
 			var modelSize = _queuedSize.IsZero ? Element.Bounds.Size : _queuedSize;
-			PageController.ContainerArea = 
-				new Rectangle(0, toolbar.Hidden ? 0 : toolbar.Frame.Height, modelSize.Width, modelSize.Height - trueBottom);
+			if (!NavigationPage.GetHasNavigationBar(Current))
```

Comment:
If SetHasNavigationBar is invoked after pushing page that content doesn't change size

![toolbar-bug](https://cloud.githubusercontent.com/assets/1790504/19310627/17e8f9aa-909c-11e6-9433-90568230fd4d.gif)